libvcx/src/api/vcx.rs

```diff
@@ -575,18 +610,31 @@ mod tests {
     #[cfg(feature = "pool_tests")]
     #[test]
     fn test_open_pool_fails_if_genesis_file_is_invalid() {
-        let _setup = SetupWallet::init();
+        let _setup = SetupDefaults::init();
+        let pool_name = format!("invalidpool_{}", uuid::Uuid::new_v4().to_string());
 
         // Write invalid genesis.txn
         let _genesis_transactions = TempFile::create_with_data(utils::constants::GENESIS_PATH, "{}");
```

Comment:
There should not be an underscore anymore since the variable is used now

Reply:
`_setup` is not accessed in this test

Reply:
I guess I must have missee